src/Entity/Activity/ExternalRegistrant.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Entity\Activity;
+
+use App\Entity\Security\ContactInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Overblog\GraphQLBundle\Annotation as GQL;
+
+/**
+ * @ORM\Embeddable
+ * @GQL\Type
+ * @GQL\Description("A registrant for an activity.")
+ */
+class ExternalRegistrant implements ContactInterface
```

Comment:
Because we don't enforce a non-null value: for example, when `new ExternalRegistrant()` is called for a form, it won't have a value necessarily on creation, but only after the form has been filled out. This means that the value is null-initiliazed, and the type hints indicate that.

I do agree that the column definition should be non-null, and will update that accordingly.

Reply:
I've found out that an Embeddable cannot be set to null; in a sense, it's more comparable to a trait. This was the reason these values were columns were set as nullable.

Reply:
still dislike it, but if we can't really change it I guess we'll have to deal. thanks for the clarification.